### DISCUSSION TITLE
Rename the default branch of morphir repos to main

### DISCUSSION THREAD

**Discussion by AttilaMihaly** [finos/morphir#62] [Ideas]
@maoo I tried renaming the default branch to `main` in the morphir repos but I don't seem to have permission to do it. Can you either grant me permission or rename it yourself? Here's some info on the process: https://github.com/github/renaming

**Comment by maoo** [comment]
Hi @AttilaMihaly - I made you admin temporarily, so you can manage the switch with the right timing and sync up with the rest of the team.

Please let me know when it's all done and I'll revoke your Admin permissions.

Thank you!

**Reply by AttilaMihaly** [reply]
Thank you. I should be able to do it today. I just want to wait for NY to get in so that I can confirm if everyone is OK with it.

**Reply by AttilaMihaly** [reply]
Hey @maoo, I'm done renaming the main repo but I want to do it for all of them. Can you make me admin for the following temporarily:
- morphir-elm
- morphir-examples
- morphir-service
- morphir-jvm
- morphir-dotnet
- morphir-bosque

**Reply by maoo** [reply]
Done!

**Reply by AttilaMihaly** [reply]
Thanks. I renamed most of them except for 2. See below.

@DamianReeves, I renamed the default branch to `main` in all morphir repos except for morphir-jvm and morphir-dotnet where you already created one. What should we do we those? Should I just drop `master` and make `main` the default? GitHub actions still seem to go from `master`.

**Reply by maoo** [reply]
@AttilaMihaly @DamianReeves - any update on this? Can I revoke admin permissions?
